.github/workflows/continuous-integration.yaml

```diff
@@ -1,10 +1,6 @@
 name: Run Tests using Docker
 
-on:
-  push:
-    branches: [ github-action ]
-  pull_request:
-    branches: [ master ]
```

Comment:
@ned21 is there any reason why you blocked gh actions of running on all branches?

Witht this setup, you cannot run gh actions on your own fork? I've made it general.

Reply:
This was my first GH action so I think it was out of an abundance of caution not to break anything or annoy anyone until it was working! Thanks for updating it now.